app/Models/InviteKey.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class InviteKey extends Model
+{
+    use HasFactory;
```

Comment:
Als iets geen Factory heeft en niet nodig gaat hebben in de toekomst (Ik denk dat dat hier het geval is) kan dit weggelaten worden. (Hetzelfde geld dan natuurlijk ook voor de `use` bovenaan in de klasse.